Specs/DateTime/English/MergedParser.json

```diff
@@ -2380,4 +2380,574 @@
+  "Context": {
+    "ReferenceDateTime": "2016-11-07T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "before or on 1/1/2016",
+      "Type": "datetimeV2.daterange",
+      "Value":{
+        "values":[
+          {
+            "timex":"2016-01-01",
+            "Mod":"until",
+            "type":"daterange",
+            "end":"2016-01-01"
+          }
+        ]
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "He will leave before 1/1/2016",
+  "Context": {
+    "ReferenceDateTime": "2016-11-07T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "before 1/1/2016",
+      "Type": "datetimeV2.daterange",
+      "Value":{
+        "values":[
+          {
+            "timex":"2016-01-01",
+            "Mod":"before",
+            "type":"daterange",
+            "end":"2016-01-01"
+          }
+        ]
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "This task will complete on or earlier than 1/1/2016",
+  "Context": {
+    "ReferenceDateTime": "2016-11-07T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "on or earlier than 1/1/2016",
+      "Type": "datetimeV2.daterange",
+      "Value":{
+        "values":[
+          {
+            "timex":"2016-01-01",
+            "Mod":"until",
+            "type":"daterange",
+            "end":"2016-01-01" 
+          }
+        ]
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "This task will complete on 1/1/2016",
+  "Context": {
+    "ReferenceDateTime": "2016-11-07T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "on 1/1/2016",
+      "Type": "datetimeV2.daterange",
+      "Value":{
+        "values":[
+          {
+            "timex":"2016-01-01",
+            "type":"date",
+            "value":"2016-01-01" 
+          }
+        ]
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "This task will complete earlier than 1/1/2016",
+  "Context": {
+    "ReferenceDateTime": "2016-11-07T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "earlier than 1/1/2016",
+      "Type": "datetimeV2.daterange",
+      "Value":{
+        "values":[
+          {
+            "timex":"2016-01-01",
+            "Mod":"before",
+            "type":"daterange",
+            "end":"2016-01-01" 
+          }
+        ]
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "This task will complete before or in Feb 2018",
+  "Context": {
+    "ReferenceDateTime": "2016-11-07T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "before or in Feb 2018",
+      "Type": "datetimeV2.daterange",
+      "Value":{
+        "values":[
+          {
+            "timex":"2018-02",
+            "Mod":"until",

```

Comment:
This case seems very weird. Mod is saying the end is inclusive, but it shouldn't include 2018-03-01 as that is not in Feb 2018.